src/Configuration/src/Kubernetes.ServiceBinding/LegacyConnectorsPostProcessors.cs

```diff
@@ -100,3 +102,59 @@ public void PostProcessConfiguration(PostProcessorConfigurationProvider provider
             });
     }
 }
+
+internal sealed class RabbitMQLegacyConnectorPostProcessor : IConfigurationPostProcessor
+{
+    internal const string BindingTypeKey = "rabbitmq";
+
+    public void PostProcessConfiguration(PostProcessorConfigurationProvider provider, IDictionary<string, string> configData)
+    {
+        if (!provider.IsBindingTypeEnabled(BindingTypeKey))
+        {
+            return;
+        }
+
+        configData.Filter(ServiceBindingConfigurationProvider.KubernetesBindingsPrefix, ServiceBindingConfigurationProvider.TypeKey, BindingTypeKey).ForEach(
+            bindingNameKey =>
+            {
+                // Spring -> spring.rabbitmq....
+                // Steeltoe -> rabbitmq:client:....
+                var mapper = new ServiceBindingMapper(configData, bindingNameKey, "rabbitmq", "client");
+                mapper.MapFromTo("addresses", "uri");
+                mapper.MapFromTo("host", "server");
+                mapper.MapFromTo("password", "password");
+                mapper.MapFromTo("port", "port");
+                mapper.MapFromTo("username", "username");
+                mapper.MapFromTo("virtual-host", "virtualhost");
+            });
+    }
+}
+
+internal sealed class RedisLegacyConnectorPostProcessor : IConfigurationPostProcessor
+{
+    internal const string BindingTypeKey = "redis";
+
+    public void PostProcessConfiguration(PostProcessorConfigurationProvider provider, IDictionary<string, string> configData)
+    {
+        if (!provider.IsBindingTypeEnabled(BindingTypeKey))
+        {
+            return;
+        }
+
+        configData.Filter(ServiceBindingConfigurationProvider.KubernetesBindingsPrefix, ServiceBindingConfigurationProvider.TypeKey, BindingTypeKey).ForEach(
+            bindingNameKey =>
+            {
+                var mapper = new ServiceBindingMapper(configData, bindingNameKey, "redis", "client");
+
+                // Mapping from Kubernetes secrets to driver-specific connection string parameters.
+                // At the time of writing (November 2023), there's no complete official documentation for the available secrets. Some pointers:
+                // - Generic secrets: https://github.com/servicebinding/spec#well-known-secret-entries
+                // - Input keys used at https://github.com/spring-cloud/spring-cloud-bindings/blob/main/spring-cloud-bindings/src/main/java/org/springframework/cloud/bindings/boot/RedisBindingsPropertiesProcessor.java
+                mapper.MapFromTo("host", "host");
+                mapper.MapFromTo("port", "port");
+                mapper.MapFromTo("ssl", "ssl");
+                mapper.MapFromTo("password", "password");
+                mapper.MapFromTo("client-name", "name");
```

Comment:
I suspect this needs to map to `ClientName` at https://github.com/SteeltoeOSS/Steeltoe/blob/release/3.2/src/Connectors/src/ConnectorBase/Redis/RedisCacheConnectorOptions.cs#L52.

This should ultimately result in the `name` connection string parameter [here](https://github.com/SteeltoeOSS/Steeltoe/blob/release/3.2/src/Connectors/src/ConnectorBase/Redis/RedisCacheConnectorOptions.cs#L110), but that's not what these legacy connectors target.